dashboard/lib/services/lti/nrps_response_validator.rb

```diff
@@ -0,0 +1,52 @@
+module Services
+  module Lti
+    class NRPSResponseValidator < Services::Base
+      REQUIRED_CUSTOM_FIELDS = %i[section_ids section_names].freeze
```

Comment:
I wonder if we should also a¡validate the `context_id` the same way you have done it here.

Reply:
The task states that we need to handle only these two fields

Reply:
This PR is specifically for NRPS, but that doesn't mean we can't open a ticket later for an ID token validator.